src/Microsoft.Azure.Relay/HybridHttpConnection.cs

```diff
@@ -254,7 +254,8 @@ async Task EnsureRendezvousAsync(CancellationToken cancelToken)
             if (this.rendezvousWebSocket == null)
             {                
                 RelayEventSource.Log.HybridHttpCreatingRendezvousConnection(this.TrackingContext);
-                var clientWebSocket = ClientWebSocketFactory.Create(this.listener.UseBuiltInClientWebSocket);
+                var clientWebSocket = ClientWebSocketFactory.Create(this.listener.UseBuiltInClientWebSocket,
```

Comment:
>Create [](start = 61, length = 6)

Style: All parameters on a single line or all on their own line:

Use
ClientWebSocketFactory.Create(\r\n
    this.listener.UseBuiltInClientWebSocket, this.listener.CustomClientWebSocketFactory);

or 

ClientWebSocketFactory.Create(\r\n
    this.listener.UseBuiltInClientWebSocket,\r\n
    this.listener.CustomClientWebSocketFactory);